can_tools/scrapers/official/AZ/az_vaccine.py

```diff
@@ -1,3 +1,8 @@
+from can_tools.scrapers.variables import (
+    FULLY_VACCINATED_ALL,
+    INITIATING_VACCINATIONS_ALL,
+    TOTAL_DOSES_ADMINISTERED_ALL,
+)
```

Comment:
Nit, but I'd advocate for importing like `from can_tools.scrapers import variables` and then referring to them by `variables.FULLY_VACCINATED_ALL` etc. 

https://google.github.io/styleguide/pyguide.html#22-imports has a good explanation - I like that it gives more context to where some constants are coming from - and you don't have to add a new import for every new variable you use

Reply:
Makes sense